xbmc/powermanagement/PowerManager.cpp

```diff
@@ -298,6 +305,14 @@ void CPowerManager::OnWake()
   g_weatherManager.Refresh();
 
   CServiceBroker::GetPVRManager().OnWake();
+  //TODO set Stack if its missing?
+  CLog::Log(LOGDEBUG, "CPowerManager::OnWake stack size %i", g_application.CurrentStack().Size());
+  //TODO set playlist because play file uses getPlaylist?
+
+  //Don't resume player if we don't get an active file during onsleep
+  if(!m_lastFileItem->IsEmpty())
+    g_application.PlayFile(*m_lastFileItem, "");
```

Comment:
this is wrong. start player if it was playing on sleep. if player was not playing, fileItem is irrelevent